components/network.rst

```diff
@@ -0,0 +1,22 @@
+Network component
+=================
+
+.. seo::
+    :description:
+    :image: network-wifi.svg
+    :keywords: Network, WiFi, WLAN, Ethernet, ESP32
+
+The network component is a global configuration for all types of 
+networks (WiFi, Ethernet).
+
+.. code-block:: yaml
+
+    # Example configuration
+    network:
+        enable_ipv6: true
+        
+Configuration variables:
+------------------------
+
+- **enable_ipv6** (*Optional*, boolean): Enables IPv6 support. Defaults to ``false``.
```

Comment:
This should say it requires the ESP-IDF framework.

Reply:
Good catch! Updated it now.